### ISSUE TITLE
I could help out a bit

### ISSUE THREAD

**Issue by Zowie** [formly-js/angular-formly-templates-material#8]
I'd like to help :) Don't have too much time but I'm more than willing to try. It'd be nice to have some info to get me started though.

**Comment by kentcdodds** [comment]
Hi @Zowie. Are you using angular-formly and material design in any projects currently? I'm happy to have you work on this repo, but unless you're doing that I'm skeptical that you'd be able to do so effectively (see [this](https://medium.com/@kentcdodds/open-source-stamina-dafd063f9932)).

Either way, I think I'd prefer that you build it on your own repository and once you've finished the first iteration we can take a look at transferring it to this repo. How does that sound?

**Comment by Zowie** [comment]
Hey Kent, I just started using it in one of my projects but your skepticism is probably valid. Sounds like a good idea, I'll set up a fork and start experimenting to find things worth working on, are there any specific things that can be done so I can look at those first?

**Comment by kentcdodds** [comment]
You could look at some of the other [plugin libraries](http://docs.angular-formly.com/page/plugins) and see how they're implemented. It's pretty much up to you since it's your project :-)

**Comment by kamilkisiela** [comment]
@kentcdodds I've done some work [here][meteor-angular-formly-templates-material]. Most of staff is done. There are also basic tests for each field and wrapper. 

Would you be interested? I could migrate it here but it's a lot of work so i need to know.

[meteor-angular-formly-templates-material]: https://github.com/wieldo/angular-formly-templates-material

**Comment by kentcdodds** [comment]
Hi @kamilkisiela! I'm happy with whatever you want to do. If you want to move ownership of your repo to the formly-js repo, it's [fairly simple](https://help.github.com/articles/transferring-a-repository/#transferring-from-a-user-to-an-organization) and we've done it for a few other repositories. This could help with discoverability. But again, it's all yours so it's up to you. (You would of course be added to the angular-formly team and have full access to this repo).

Either way, I'll add your library to the plugins page! Thanks!

**Comment by kamilkisiela** [comment]
@kentcdodds I see. I will transfer my repo ownership to formly-js, but not until saturday, have a lot of work right now. I will message you on gitter.

**Comment by kentcdodds** [comment]
I'm going to be deleting this repository in favor of @kamilkisiela's. Stand by...
